README.md

```diff
@@ -82,6 +82,10 @@ For all bug and feature requests please [file a GitHub issue](https://github.com
 
 Discussions are done using [Github Discussions](https://github.com/containers/podman-desktop/discussions/).
 
+Join #podman-desktop on the [Kubernetes Slack](https://communityinviter.com/apps/kubernetes/community).
```

Comment:
We should probably have the direct link to the channel as well (if people are already on the instance) https://app.slack.com/client/T09NY5SBT/C04A0L7LUFM

Here it looks like we're displaying the 3rd party provider url but instead maybe we need to use the alias from kubernetes/k8s domain http://slack.k8s.io/